backend/app/utils/ai_processors.py

```
@@ -331,3 +269,118 @@ def __init__(self):
+    
+    results = {}
+    
+    try:
+        # 更新内容项状态
+        content_item.processing_status = "processing"
+        content_item.last_processed_at = datetime.utcnow()
+        session.add(content_item)
+        session.commit()
+        
+        # 调用AI分析
+        chat_service = ChatService()
```

Ruff (0.11.9)

Undefined name `ChatService`

(F821)